Trivago/Jade/Application/Framework/JadeBundle/Controller/JsonApiController.php

```diff
@@ -600,7 +642,7 @@ private function setupJsonApiBaseUrl(Request $request, $pathPartsCount)
         $baseUrl = $request->getSchemeAndHttpHost().$request->getBaseUrl().$request->getPathInfo();
 
         $baseUrlParts = explode('/', $baseUrl);
-        for ($i=0; $i<$pathPartsCount; $i++) {
+        for ($i = 0; $i < $pathPartsCount; $i ++) {
```

Comment:
$i ++ kind of looks wrong

Reply:
It is the PSR-2 code styling. I will remove this kind of styling before push changes